tests/fixtures/triggers/github-pull-request-full.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="utf-8"?>
+<project>
+  <triggers class="vector">
+    <org.jenkinsci.plugins.ghprb.GhprbTrigger>
+      <adminlist>user1
+user2</adminlist>
+      <whitelist>user3
+user4</whitelist>
+      <orgslist>org1
+org2</orgslist>
+      <whiteListLabels>label1
+label2</whiteListLabels>
+      <blackListLabels>label3
+label4</blackListLabels>
+      <excludedRegions>exclude
+region</excludedRegions>
+      <includedRegions>include
+region</includedRegions>
+      <buildDescTemplate>build description</buildDescTemplate>
+      <spec>* * * * *</spec>
+      <allowMembersOfWhitelistedOrgsAsAdmin>true</allowMembersOfWhitelistedOrgsAsAdmin>
+      <cron>* * * * *</cron>
+      <triggerPhrase>retest this please</triggerPhrase>
+      <skipBuildPhrase>no tests</skipBuildPhrase>
+      <blackListCommitAuthor>blacklist commit author</blackListCommitAuthor>
```

Comment:
I guess it's newline separated list when edited via UI, right?

Reply:
No you cannot put newlines, you can only put spaces, I have made the necessary changes to handle it.